package.json

```diff
@@ -74,14 +74,14 @@
     "@microsoft/load-themed-styles": "1.10.26",
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32830&commitId=73cdaf998b0b55630f8c301451122f4723ff836d&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|28| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1136| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|4940| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>